low-pass-filter/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "solana-low-pass-filter"
+description = "Low Pass Filter"
+version = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+publish = false
+
+[lib]
+path = "src/lib.rs"
+
+[features]
+default = []
```

Comment:
FYI by default all features are disabled https://doc.rust-lang.org/cargo/reference/features.html#the-default-feature. Does not matter here of course.